libs/external-db-postgres/src/postgres_operations.ts

```diff
@@ -12,6 +12,6 @@ export default class DatabaseOperations implements IDatabaseOperations {
     // @ts-ignore
     async validateConnection() {
         return await this.pool.query('SELECT 1').then(() => { return { valid: true } })
-                              .catch((e) => { return { valid: false, error: notThrowingTranslateErrorCodes(e) } })
+                              .catch((e) => { return { valid: false, error: notThrowingTranslateErrorCodes(e, '') } })
```

Comment:
is it needed? can we make the parameter optional and transform it to empty string if needed in `notThrowingTranslateErrorCodes` ?

Reply:
I need the collection name for the error in changing the column type, and I'd rather it not be optional, so I must put an empty string for the collection name argument. 
We did the same thing in MySQL. I can set a default value for the collection name argument ...., is it better?

Reply:
If you need the collection name only for part of the errors, it sounds like an optional param, putting default value is fine as well